src/hooks/users.ts

```diff
@@ -11,7 +11,7 @@ import { useSparkleFirestoreConnect } from "hooks/useSparkleFirestoreConnect";
 
 import { User } from "types/User";
 
-const useConnectUsers = () => {
+const useConnectVenueUsers = () => {
```

Comment:
[NB (non-blocking)] When I mentioned changing this, I was thinking we should change `storeAs` to `venueUsers` as well, since the default within the firestore library is to use the collection name (`users`) as the default value for `storeAs`. This avoids us 'shadowing' the same concepts with different meanings, and makes it more explicit/obvious what the purpose is/that we won't inadvertently break something/etc.

Reply:
Yes, I do remember this. I got you right. It's just it didn't take much time for me to change it to `venueUsers`, so I did that. I didn't forget about renaming `users` to `venueUsers` in our redux store. It's just a task for future PR

Reply:
Seems you backlogged this in https://github.com/sparkletown/internal-sparkle-issues/issues/124 👌🏻